docs/docs/100-reference/06-compute-resources.md

```diff
@@ -70,10 +70,16 @@ unrestricted            Unrestricted      512Mi            Unrestricted ComputeC
 specific                128Mi,512Mi,1Gi   128Mi            Specific ComputeClass
 ```
 
-Breaking this down, `MEMORY_DEFAULT` tells us what memory we will get if we don't specify any. `MEMORY_RANGE` tells us what memory values are available to use. If it is a range, specified with a `-` then you can use any value in that range. If it has specific values, denoted by a comma seperated list, then you can only use those values. Specifying workload classes can be done in the Acornfile (using the `class` property for containers) or at runtime (using the `--compute-class` flag). 
+Breaking this down:
 
-If you do not specify a compute class, the default compute class for the project will be used. If there is no default for the project, the default for the cluster will be used. Finally, if there is no cluster default then no compute class will be used. Depending on the compute class that is used, the memory that you specify may be in contention with its requirements. Should that happen Acorn will provide a descriptive error message to ammend any issues.
+- `DEFAULT` says if a ComputeClass is the default and will be used if no ComputeClass is defined.
+- `MEMORY_DEFAULT` tells us what memory we will get if we don't specify any. 
+- `MEMORY_RANGE` tells us what memory values are available to use and can take two forms.
+    1. `-` denotes a range and any value within it can be used
+    2. `,` denotes specific values that are the only ones allowed.
```

Comment:
```suggestion
- `DEFAULT` indicates whether a ComputeClass is the default and will be used if no ComputeClass is specified for a workload.
- `MEMORY_DEFAULT` indicates the memory that will be requested for a workload if none is specified.
- `MEMORY_RANGE` indicated what memory values are available to use, and can take two forms.
    1. `-` denotes a range and any value within it can be specified.
    2. `,` denotes specific values that are the only ones allowed.
```